contributing/telemetry.md

```diff
@@ -15,3 +18,23 @@ Additionally, if telemetry is disabled for Salesforce CLI using the `disable-tel
 - Create a telemetry service under src
 - Initialize the telemetry service on the extension's `activate` call and initialize it using `salesforce-vscode-core` telemetry service
 - Use the telemetry service where needed in the extension
+
+## Logging telemetry to a file
```

Comment:
Btw, I just noticed this slack thread https://salesforce-internal.slack.com/archives/C4LRHH0AD/p1726781465323499?thread_ts=1726780954.506789&cid=C4LRHH0AD where Pete mentions this setting for local telemetry "salesforcedx-vscode-apex.advanced": {
      "localTelemetryLogging": "true"
  },

Are we missing it in the docs?

Reply:
Wasn't aware of that one.   It's dev mod specific.  Will add it.